### ISSUE TITLE
Active Keys not working

### ISSUE THREAD

**Issue by atlanted** [BOLL7708/OpenVR2Key#44]
My active keys says unblocked but when I click a button on my quest 2 controller nothing happens.

**Comment by BOLL7708** [comment]
1. Is a game running?
2. Is the dashboard hidden?
3. Did you map any keys?

**Comment by atlanted** [comment]
I'm in steam VR home, my dashboard is hidden, and I haven't mapped any keys yet because I don't know keys to map since it doesn't show me what buttons are which.

**Comment by BOLL7708** [comment]
I you haven't mapped any buttons nothing will happen yeah, you click an input to map and you hold the key combo down that you want to get simulated, then click the button again to save it. The inputs being triggered are listed at the top when pressed.

**Comment by neo64-balls** [comment]
my active keys are not working and it says its unblocked and my dashboard is closed and when i click  a button  on my quest 2 it dose  nothing

**Comment by neo64-balls** [comment]
and steam says the action manifest for openvr2key was not found

**Comment by BOLL7708** [comment]
So it worked on one machine but not the other? If you go into the SteamVR bindings do you get actions to bind to buttons at all? If those exist, and the default bindings have been loaded, it is supposed to have each button bound and registering in the application already. 

Did you decompress the folder to the same place on both machines?

**Comment by KaiSSeRToRiuS** [comment]
It works on one machine but not the other. If I go into settings > controllers > SteamVR bindings (I don't know the exact order) I can configure actions and buttons. I have tried to assign keyboard keys to buttons in the app, and then open a txt to check and it doesn't work. I have extracted the files on both machines to the desktop. I try to run normal and as admin. In a while I will have access to my laptop. I can try to record video or take screenshot.

**Comment by KaiSSeRToRiuS** [comment]
But what a fool I've been! I hadn't realized that, you have to be inside the SteamVR room and meanwhile, touch buttons... So it does work... I hit the Oculus button, desktop button while leaving SVR open... I don't know If I have been able to explain myself, but now it works perfectly. Thank you very much and excuse my stupidity!

**Comment by BOLL7708** [comment]
I'm not entirely sure what is going on but good thing it is working 😅

**Comment by KaiSSeRToRiuS** [comment]
Yes, I don't explain myself well and also my English is a disaster. Let's see this time... For the program to work, once you open SteamVR, you must leave it as it is. I was doing like an ALT+TAB. I hit the Oculus button and went out to the desktop to see the app from inside, with the glasses and... That's where it stopped working. So just go into SVR and a) Take off your headset and set it aside and look at the screen. B) Raise the visor and look at the screen. Better explained now?

Indeed, what matters is that it works. Thanks again.

**Comment by BOLL7708** [comment]
Ah, the input is blocked when using the SteamVR dashboard, but I think you're saying it is also blocked when using the virtual desktop feature in the Oculus runtime? I run a SteamVR native headset so I have not tried this workflow, but I think this is what is going on! And English is my second language which doesn't help either, but we're making do! 🤣👍
